schema.ts

```diff
@@ -568,6 +568,26 @@ export interface StringDateTimeSchema extends BaseStringSchema<ValueHintType.Dat
   timeFormat?: string;
 }
 
+/**
+ * A schema representing a return value or object property that is provided as a string,
+ * which Coda should interpret as an image.
+ */
+ export interface StringImageSchema extends BaseStringSchema<
+  ValueHintType.ImageReference | ValueHintType.ImageAttachment> {
+  /** Instructs Coda to render this value as a Image with proper beautifulImage properties. */
```

Comment:
I wouldn't mention the beautifulImage here, as this documentation is public. Just say "Instructs Coda to render this value as an Image"?